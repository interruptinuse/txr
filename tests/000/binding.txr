@(bind x 1)
@(bind y 2)
@(rebind z (x y))
@(bind (a b) (0 1))
@(rebind (a b) (b a))
