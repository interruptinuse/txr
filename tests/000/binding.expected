x="1"
y="2"
z[0]="1"
z[1]="2"
a="1"
b="0"
